scipy/integrate/_quadrature.py

```diff
@@ -9,8 +9,7 @@
 from scipy.special import roots_legendre
 from scipy.special import gammaln, logsumexp
 from scipy._lib._util import _rng_spawn
-from scipy._lib.deprecation import (_NoValue, _deprecate_positional_args,
-                                    _deprecated)
+from scipy._lib.deprecation import (_deprecated)
```

Comment:
nit: looks a little strange

```suggestion
from scipy._lib.deprecation import _deprecated
```

Reply:
Blame ruff `--fix` ;)